src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -263,8 +263,8 @@ private static bool IsVarDesired(ITypeSymbol type, OptionSet options)
                 return options.GetOption(CSharpCodeStyleOptions.UseImplicitTypeForIntrinsicTypes).Value;
             }
 
-            // If they want "var" whenever possible, then use "var".
-            return options.GetOption(CSharpCodeStyleOptions.UseImplicitTypeWherePossible).Value;
+            // If they want "var" whenever possible, then use "var" even if the type is apparent.
+            return options.GetOption(CSharpCodeStyleOptions.UseImplicitTypeWherePossible).Value && options.GetOption(CSharpCodeStyleOptions.UseImplicitTypeWhereApparent).Value;
```

Comment:
1. wrap long lines (i.e. if this can't be seen on github, it should be wrapped)
2. I don't really understand your change.  If someone says they want implicit types when possible, then we should respect that.

Reply:
I don't understand how this is the source of hte bug.  The original issue says that the editorconfig file has `csharp_style_var_elsewhere = false:suggestion` in it.  so this should have been returning 'false', so we should not have been using 'var'.